src/Storage/Storage.Management/Models/PSLocalUser.cs

```diff
@@ -52,6 +55,8 @@ public LocalUser ParseLocalUser()
             user.HasSshPassword = this.HasSshPassword;
             user.SshAuthorizedKeys = PSSshPublicKey.ParseSshPublicKeyss(this.SshAuthorizedKeys);
             user.PermissionScopes = PSPermissionScope.ParsePermissionScopes(this.PermissionScopes);
+            user.GroupId = this.GroupId;
+            user.AllowAclAuthorization = this.AllowAclAuthorization;
```

Comment:
might be a silly question: why there are no "UserId" here?

Reply:
It's because UserId is a readonly property set by the server. When creating the local user, we're not able to set the value